collections/open-source-organizations/index.md

```diff
@@ -15,6 +15,9 @@ items:
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
  - OSGeo/osgeo
+ - Airbnb/github.com/airbnb
+ - PayPal/github.com/paypal
+ - WalmartLabs/github.com/walmartlabs
```

Comment:
These all need to be without the `github.com/` part.

Reply:
There are several organisations that don't have their github pages like PayPal, Airbnb, WalmartLabs, Evernote, Pinterest.

Reply:
@kaus02 This syntax as-is won't work. You need to omit the `github.com/`. Would you like me to reopen this so you can push a commit to address this feedback?